app/views/coronavirus_form/offer_care_qualifications.erb

```diff
@@ -54,7 +54,13 @@
           error_message: error_items("offer_care_qualifications_type"),
           value: session[:offer_care_qualifications_type],
         })
-      },
+      }
+    ]
+  } %>
+  <%= tag.p t("coronavirus_form.questions.offer_care_qualifications.care_qualifications.or"), class: "govuk-body" %>
```

Comment:
The radio button component has an [or divider](https://components.publishing.service.gov.uk/component-guide/radio/with_or_divider). Perhaps something similar could be added the checkbox component in the future? What do you think @alex-ju?

Reply:
I added the or divider to the radios component as it's an established pattern (and it makes sense to have it as a separator since the logic for radio is an or-based), but for checkboxes I'm reluctant to introduce something similar as they're and-based and introducing an Or would make users confusing. This is the reason we have this workaround in the app – to avoid making it available in other places.